client/src/utils/tone/index.ts

```diff
@@ -0,0 +1,83 @@
+import store from 'store';
+import { FlashMessageArg } from '../../components/Flash/redux';
+import { Themes } from '../../components/settings/theme';
+
+// TODO: Add more tone types
+export async function playTone(
+  state: FlashMessageArg['message'] | Themes | string
+): Promise<void> {
+  const playSound = !!store.get('fcc-sound');
+  if (playSound) {
+    const tone = await import('tone');
+    if (tone.context.state !== 'running') {
+      void tone.context.resume();
+    }
+    let player;
+    switch (state) {
+      case 'flash.incomplete-steps':
+        player = new tone.Player(
+          'https://campfire-mode.freecodecamp.org/try-again.mp3'
+        ).toDestination();
+        player.autostart = playSound;
```

Comment:
```suggestion
        player.autostart = true;
```
`playSound` is `true` at this point.